src/go/rpk/pkg/cli/debug/bundle/bundle_k8s_linux.go

```diff
@@ -57,10 +59,11 @@ func executeK8SBundle(ctx context.Context, bp bundleParams) error {
 	defer w.Close()
 
 	ps := &stepParams{
-		fs:       bp.fs,
-		w:        w,
-		timeout:  bp.timeout,
-		fileRoot: strings.TrimSuffix(filepath.Base(bp.path), ".zip"),
+		fs:        bp.fs,
+		w:         w,
+		timeout:   bp.timeout,
+		fileRoot:  strings.TrimSuffix(filepath.Base(bp.path), ".zip"),
+		sharedBuf: make([]byte, 32*1024),
```

Comment:
[nit]: what `32*1024` do you think having a variable that explain the why via name? like limit-buffer